manual/src/main/asciidoc/whats-new.adoc

```diff
@@ -16,14 +16,130 @@
 Apache Unomi 3 is a new release focused on integrations of the client to support elasticsearch version 9.
 It also include the upgrade of the Karaf version.
 
-==== Elasticsearch client upgrade
+=== Elasticsearch client upgrade
 
 The official client for Elasticsearch has been added to Apache Unomi in version 3.0 in order to replace the old rest-client which
 is not supported anymore.
 
 The documentation of the client can be found here: https://www.elastic.co/docs/reference/elasticsearch/clients/java
 
-==== Karaf upgrade
+=== Migrate from Elasticsearch 7 to Elasticsearch 9
+
+You can use the *remote reindex* API to upgrade directly from Elasticsearch 7 to Elasticsearch 9. This approach runs both clusters in parallel and uses Elasticsearch's remote reindex feature.
+
+This upgrade relies on a script that was specifically built taking jCustomer into consideration. If you are sharing the Elasticsearch instance with other projects, it might need to be adjusted.
```

Comment:
Not sure people will know what jCustomer is, maybe we should just say a custom Unomi application instead ?